FactoryWeather/App/VIP Search/SearchViewController.swift

```diff
@@ -0,0 +1,152 @@
+//
+//  SearchViewController.swift
+//  FactoryWeather
+//
+//  Created by Matej Korman on 14/01/2019.
+//  Copyright © 2019 Matej Korman. All rights reserved.
+//
+
+import UIKit
+import SnapKit
+import Promises
+
+protocol SearchDisplayLogic: class {
+    func displayLocations(_ locations: [Location])
+    func displayError(_ error: Error)
+}
+
+class SearchViewController: UIViewController {
+    var interactor: SearchBusinessLogic?
+    var router: SearchRoutingLogic?
+    private let contentView: SearchContentView
+    private let activityIndicatorView = UIActivityIndicatorView.autolayoutView()
```

Comment:
Just a note, `activityIndicatorView` should not be located in the `view controller`, but the `content view`.

Reply:
Yep, we discussed that on another PR, but agreed to leave it as is. (This is leftover from old code and could be improved, but you said the main task of this sprint is to learn the architecture)

Reply:
Roger.